templates/lifecycle.md

```diff
@@ -63,15 +63,17 @@ Additionally, for **building blocks**:
 Additionally, for **building blocks**:
 
 - [ ] Conformance tests updated to match any API changes that have been made
-- [ ] Conformance tests exercise both positive and negative cases 
-- [ ] Minimum of N (three?) implementations of this building block 
-- [ ] Certification tests for implementations 
+- [ ] Conformance tests exercise both positive and negative cases
+- [ ] Minimum of N (three?) implementations of this building block
```

Comment:
Should we keep this three or is it a TBD?

Reply:
Good call out, this should be decided. IMO 3 is reasonable.